### ISSUE TITLE
[BUG] PKMODEL does not allow covariate params

### ISSUE THREAD

**Issue by dpastoor** [metrumresearchgroup/mrgsolve#39]
Works fine:

```
test <- '
$PARAM CL = 0.3, VC = 1.5
$CMT CENT
$PKMODEL ncmt=1, trans=11
$MAIN
double CLi = CL*exp(ETA(1));
double Vi = VC*exp(ETA(2));
$OMEGA
0.1
0.04
$CAPTURE CLi Vi
'
```

however adding a covariate param still compiles fine, but on execution

```
fails <- '
$PARAM CL = 0.3, VC = 1.5, wt = 5
$CMT CENT
$PKMODEL ncmt=1, trans=11
$MAIN
double CLi = CL*exp(ETA(1))*wt/5;
double Vi = VC*exp(ETA(2));
$OMEGA
0.1
0.04
$CAPTURE CLi Vi wt
'
```

get the following error

```
Error: A PK parameter has an invalid value.
```

**Comment by kylebmetrum** [comment]
The `fails` code is working for me.  The error is generated whenever CL or V are not positive.  Trying to think of what's going on here.

After you compile with `fails`, what do you get for `mod@advan` and `mod@trans`?  For a while, I didn't have that quite worked out correctly.  It could also be a mistake in the code so the it is expecting some other parameter to have non-zero value.

**Comment by dpastoor** [comment]
your comment plus digging into where the error gets thrown led me to the problem - I had some row values where there had been an NA value coerced to 0 for wt, which led to a 0 V.

**Comment by dpastoor** [comment]
addressed in #40
